tests/privacy_simple_network.py

```diff
@@ -122,20 +108,37 @@ def publishContract(account, file, waitForTransBlock=False):
     participants = [x for x in producers]
     nonParticipants = [x for x in apiNodes]
 
-    def security_group(nodeNums):
-        action = None
-        for nodeNum in nodeNums:
-            if action is None:
-                action = '[['
-            else:
-                action += ','
-            action += '"{}"'.format(Node.participantName(nodeNum))
-        action += ']]'
-
-        Utils.Print("adding {} to the security group".format(action))
-        trans = producers[0].pushMessage(cluster.eosioAccount.name, "add", action, "--permission eosio@active")
-        Utils.Print("add trans: {}".format(json.dumps(trans, indent=4, sort_keys=True)))
-        trans = producers[0].pushMessage(cluster.eosioAccount.name, "publish", "[0]", "--permission eosio@active")
+    # this is passed to limit the number of add/remove table entries are processed, but using it here to keep from getting duplicate transactions
+    publishProcessNum = 20
+    def security_group(addNodeNums=[], removeNodeNums=[]):
+        def createAction(nodeNums):
```

Comment:
```
def createAction(nodeNums):
    return None if len(nodeNums) == 0 else '[[{}]]'.format(','.join(['"{}"'.format(Node.participantName(nodeNum)) for nodeNum in nodeNums]))
```

Reply:
those one liners are hard to read.
we can have at least something like that:
```
def createAction(nodeNums):
     if len(nodeNums) == 0 
         return None
     return '[[{}]]'.format(','.join(['"{}"'.format(Node.participantName(n)) for n in nodeNums]))
 ```